### DISCUSSION TITLE
x-data内的以"a.b.c"格式为key的对象被转换

### DISCUSSION THREAD

**Discussion by Alex-xd** [alibaba/formily#3490] [Q&A 2.x] [answered]
schema：
![image](https://user-images.githubusercontent.com/11256006/197743798-c88d80fb-98bf-4455-b005-bcdf8c918b29.png)

实际取出的
![image](https://user-images.githubusercontent.com/11256006/197743716-d46e7045-2346-4a61-a118-5edf49095a87.png)

该转换行为不符合期望，有没有用户可以手动控制的口子？

**Comment by Alex-xd** [comment, marked as answer]
目前我临时用lodash的get去取值

**Comment by janryWang** [comment]
加个转义 `a\\.\\.b\\.c`
